### PR TITLE
DEVELOPER-1357 - Update Home Page Hero: Add Vert.X 3 release announce…

### PR DESCRIPTION
**Pull request by Dantheman720**
…ment

### CODE REVIEW

File: index.html.slim [hunk 1]
```diff
@@ -29,9 +29,9 @@ description: The simple, modern & productive way to build apps and infrastructur
           img.mobile(src="http://static.jboss.org/images/hero/hero_mobile_480x500_devnation_06112015.jpg" alt="DevNation: Watch the keynotes, live and on demand")
           img.desktop(src="http://static.jboss.org/images/hero/hero_desktop_1175x400_devnation_06112015.jpg" alt="DevNation: Watch the keynotes, live and on demand")
       .large-24.slide
-        a( href="https://www.redhat.com/apps/store/middleware/jboss_enterprise_application_platform.html")
-          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_480x500_eapwebstore_051215.jpg") alt="")
-          img.desktop(src=cdn("http://static.jboss.org/images/hero/hero_desktop_1175x400_eapwebstore_051215.jpg") alt="")
+        a( href="http://vertx.io/")
+          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_480x500_vertx3_06252015.jpg") alt="")
```
Comment (paulrobinson) [inline index.html.slim]: You need to put alt tag values in. This messes up SEO if we miss them off. Use the text from the image.

**Comment by Dantheman720** [comment]
Adding now

**Comment by Dantheman720** [comment]
retest this please

File: index.html.slim [hunk 2]
```diff
@@ -29,13 +29,13 @@ description: The simple, modern & productive way to build apps and infrastructur
           img.mobile(src="http://static.jboss.org/images/hero/hero_mobile_480x500_devnation_06112015.jpg" alt="DevNation: Watch the keynotes, live and on demand")
           img.desktop(src="http://static.jboss.org/images/hero/hero_desktop_1175x400_devnation_06112015.jpg" alt="DevNation: Watch the keynotes, live and on demand")
       .large-24.slide
-        a( href="https://www.redhat.com/apps/store/middleware/jboss_enterprise_application_platform.html")
-          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_480x500_eapwebstore_051215.jpg") alt="")
-          img.desktop(src=cdn("http://static.jboss.org/images/hero/hero_desktop_1175x400_eapwebstore_051215.jpg") alt="")
+        a( href="http://vertx.io/")
+          img.mobile(src=cdn("http://static.jboss.org/images/hero/hero_mobile_480x500_vertx3_06252015.jpg") alt="A tool-kit for building reactive applications on the JVM"")
```
Comment (paulrobinson) [inline index.html.slim]: This line and the one below has tow double quotes at the end. This is failing the build.

**Comment by Dantheman720** [comment]
Alright, I removed the extra quotes...how frustrating.

**Pull request merged**